package.json

```diff
@@ -3,6 +3,7 @@
   "version": "5.0.2",
   "description": "React tags is a fantastically simple tagging component for your React projects",
   "main": "dist-modules/ReactTags.js",
+  "sideEffects": false,
```

Comment:
@tabrath currently we are at webpack `3.11.0` so I guess this flag won't be working .  Can you remove this from this pr and give a PR with updating webpack and also adding this flag ?